src/tests/class_tests/openms/source/SpectrumExtractor_test.cpp

```diff
@@ -443,6 +437,147 @@ START_SECTION(pickSpectrum())
 }
 END_SECTION
 
+START_SECTION(annotateSpectrum())
+{
+  MzMLFile mzml;
+  PeakMap experiment;
+  TargetedExperiment targeted_exp;
+  TransitionTSVReader tsv_reader;
+
+  // targeting a particular mz and rt
+  //double target_mz = 177.0;
+  //double target_rt = 840.0;
+
+  ptr->setRTWindow(30);
+  ptr->setMZTolerance(0.1);
+  ptr->setGaussWidth(0.25);
+  ptr->setUseGauss(true);
+
+  // mzml.getOptions().setRTRange(DRange<1>(target_rt - ptr->getRTWindow() / 2.0, target_rt + ptr->getRTWindow() / 2.0));
+  // mzml.getOptions().setRTRange(DRange<1>(0.0, 856));
+  // mzml.getOptions().setMZRange(DRange<1>(target_mz - ptr->getMZTolerance(), target_mz + ptr->getMZTolerance()));
+  // mzml.getOptions().setMZRange(DRange<1>(0.0, 300.0));
+
+  // TODO remove the following path, use different way to load data
+  mzml.load("/home/pasdom/Downloads/douglas_files/13C1.mzML", experiment);
+  std::vector<MSSpectrum> spectra = experiment.getSpectra();
+  tsv_reader.convertTSVToTargetedExperiment("/home/pasdom/Downloads/douglas_files/13CFlux_TraML.csv", FileTypes::TRAML/*FileTypes::CSV*/, targeted_exp);
```

Comment:
If we whittle the 13C1.mzML file down, we may be able to whittle the TraML file for this test just to the select transitions.  That way we would not even need to read in the whole file, but manually build the class from the select transition.

Reply:
Opened an issue about it: https://github.com/biosustain/OpenMS/issues/19